src/client/src/Web/index.tsx

```diff
@@ -33,142 +40,123 @@ const TornOut = ({ children }: WithChildren) => (
   </TearOutContext.Provider>
 )
 
+export const TileWrapper = () => {
```

Comment:
could you extract this to its own module - suggest in `src/client/src/App/LiveRates/Tile` or perhaps `TearOut` beneath that

Reply:
good idea .. wonder why that was not extracted earlier